### PR TITLE
Cunvoas

### PR DESCRIPTION
**Pull request by cunvoas**
Add PostgreSQL 

I let :
SQL, stored procs, exec plan, backup/restore

### CODE REVIEW

File: postgresql/postgresql.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Indexes
+When you design your database, each primary key and foreign key have an automatic index. But in som cases, you nedd to add another ones for performance.
+
+    By example, we need to query with a WHERE clause on two columns (begin_date and end_date) but this query is slow.
+    
+    CREATE INDEX idx_people 
+        ON people(begin_date, end_date) 
+        USING BTREE;
+
```
Comment (PiotrFLEURY) [inline postgresql/postgresql.md]:
```suggestion
    ```sql
    CREATE INDEX idx_people 
        ON people(begin_date, end_date) 
        USING BTREE;
    ```
```

File: postgresql/postgresql.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        id         long not null,
+        name       string not null,
+        country    string not null
+        ...
+    ) PARTITION BY LIST (country);
+      
+    CREATE TABLE people_eu 
+        PARTITION OF people
+        FOR VALUES IN ('FR', 'ES', 'IT', 'DE', 'NL' , ...., 'PL');
+
```
Comment (PiotrFLEURY) [inline postgresql/postgresql.md]:
```suggestion
    ```sql
    CREATE TABLE people (
        id         long not null,
        name       string not null,
        country    string not null
        ...
    ) PARTITION BY LIST (country);
      
    CREATE TABLE people_eu 
        PARTITION OF people
        FOR VALUES IN ('FR', 'ES', 'IT', 'DE', 'NL' , ...., 'PL');
    ```
```

**Review by PiotrFLEURY — COMMENTED** [review]
You can englobe sql scripts in code blocks for better readability

**Comment by cunvoas** [comment]
ok with mods

**Review by PiotrFLEURY — APPROVED** [review]

**Pull request merged**